speed_games/demo_website/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "glitch-hello-node",
+  "version": "0.1.0",
+  "description": "A simple Node app built on fastify, instantly up and running. Built to be remixed on Glitch.com.",
+  "main": "server.js",
+  "scripts": {
+    "start": "node server.js"
+  },
+  "dependencies": {
+    "fastify": "^4.12.0",
+    "@fastify/static": "^6.8.0",
+    "handlebars": "^4.7.7",
+    "@fastify/view": "^7.4.1",
+    "@fastify/compress": "^6.2.0"
+  },
+  "engines": {
+    "node": "14.x"
+  },
+  "repository": {
+    "url": "https://glitch.com/edit/#!/glitch-hello-node"
```

Comment:
Looks like a template placeholder, if possible please adjust the name or remove it, if not needed.